src/main/resources/export/java/RobotTemplate.java

```diff
@@ -14,7 +14,7 @@
  * The VM is configured to automatically run this class, and to call the
  * functions corresponding to each mode, as described in the TimedRobot
  * documentation. If you change the name of this class or the package after
- * creating this project, you must also update the build.properties file in 
+ * creating this project, you must also update the build.properties file in
```

Comment:
`build.properties` was with the old (_old_) Ant build system. It would be nice to update this but not necessarily in this PR